offlinevalidator/offlinevalidator_test.go

```diff
@@ -336,8 +337,18 @@ func TestInvalidMapProperty(t *testing.T) {
 	assert.False(t, validateResources(resources, &spec, &sink, deadProp, deadRes), "This resource should be valid")
 }
 
-func createResourceWithNestedProperties(resourceType string, propertyName string, nestedPropertyValue map[string]interface{}) template.Resource {
+func TestFindingAllowedValues(t *testing.T) {
```

Comment:
`findingAllowedValues` is quite complicated function. Please provide more tests to test more than one path.